src/client/game/game-actions.js

```diff
@@ -177,3 +183,43 @@ export function gameFinished() {
 		type: GAME_FINISHED
 	}
 }
+
+export function nextRoundType(type) {
```

Comment:
This file really needs testing. I also wonder whether game-actions could be split into number-round-actions and letters-round-actions?